### ISSUE TITLE
POI Alarm feature

### ISSUE THREAD

**Issue by sonora** [osmandapp/Osmand#7834]
In light if #7829, we may want to think about a POI Alarm feature. I know of users who activate certain POI categories when on a trip and if they happen to come within a set vicinity of a POI they want to be notified to have a chance to check it out.

**Comment by pebogufi** [comment]
And this would also be very helpful if it works not only in navigation mode but also in "navigation-prepared". So one could use it as "approaching alarm". Additioal to that if it works at approaching to a favourite.

**Comment by sonora** [comment]
Yes, I am even thinking it should work regardles of any 'route calculated' state or 'turn-by-turn guidance active' mode, just a simple 'Vicinity alert'.

**Comment by vshcherb** [comment]
It is not clear whether it is a regression or not, if it is a regression that would imply on priority of that issue as well

**Comment by vshcherb** [comment]
We would need to redesign how it works now cause this feature is tighten with Navigation Route
